lib/istio/addon/components/gateway-server/template.hbs

```diff
@@ -0,0 +1,274 @@
+          class="btn bg-transparent text-small vertical-middle"
+          {{action "removeServer" server}}
+        >
+          {{t "cruGateway.servers.removeServerLabel"}}
+        </button>
+      </div>
+    </div>
+  {{/if}}
+
+  <div class="row">
+    <div class="col span-4">
+      <label
+        class="acc-label"
+        for="{{concat elementId "-input-port-number"}}"
+      >
+        {{t "cruGateway.port.number.label"}}
+        {{#if editing}}
+          {{field-required}}
+        {{/if}}
+      </label>
+      {{#input-or-display
+        editable=editing
+        value=server.port.number
+      }}
+        {{input-integer
+          min=1
+          max=65535
+          id=(concat elementId "-input-port-number")
+          value=server.port.number
+          classNames="form-control"
+          placeholder=(t "cruGateway.port.number.placeholder")
+        }}
+      {{/input-or-display}}
+    </div>
+
+    <div class="col span-4">
+      <label
+        class="acc-label"
+        for="{{concat elementId "-input-port-name"}}"
+      >
+        {{t "cruGateway.port.name.label"}}
+        {{#if editing}}
+          {{field-required}}
+        {{/if}}
+      </label>
+      {{#input-or-display
+        editable=editing
+        value=server.port.name
+      }}
+        {{input
+          type="text"
+          id=(concat elementId "-input-port-name")
+          value=server.port.name
+          classNames="form-control"
+          placeholder=(t "cruGateway.port.name.placeholder")
+        }}
+      {{/input-or-display}}
+    </div>
+
+    <div class="col span-4">
+      <label class="acc-label">
+        {{t "cruGateway.port.protocol.label"}}
+        {{#if editing}}
+          {{field-required}}
+        {{/if}}
+      </label>
+      {{#input-or-display
+        editable=editing
+        value=server.port.protocol
+      }}
+        {{new-select
+          classNames="form-control"
+          content=protocols
+          value=server.port.protocol
+        }}
+      {{/input-or-display}}
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col span-12">
+      {{#form-value-array
+        editing=editing
+        valueLabel="cruGateway.hosts.label"
+        addActionLabel="cruGateway.hosts.add"
+        changed=(action "setHosts")
+        addButtonClass="btn bg-link icon-btn"
+        initialValues=server.hosts
+        showProTip=false
+        required=true
+        as |c row|
+      }}
+        {{#input-or-display
+          editable=editing
+          value=row.value
+        }}
+          {{input
+            class="form-control input-sm"
+            type="text"
+            value=row.value
+            placeholder=(t "cruGateway.hosts.placeholder")
+          }}
+        {{/input-or-display}}
+      {{/form-value-array}}
+    </div>
+  </div>
+
+  {{#advanced-section advanced=advanced}}
+    <hr class="mt-20"/>
+    <div class="row">
+      <div class="col span-4">
+        <label class="acc-label">
```

Comment:
Do we have issues with using the for/id semantics with `input-or-display`? If not i'd like to add the accessibility here and the other places we can in this file.